### PR TITLE
BL Recovery Password Hints Commit

### PR DESCRIPTION
**Pull request by rafals2**
Added explanation of recovery password mechanics (password hints and custom URL)

### CODE REVIEW

File: windows/security/information-protection/bitlocker/bitlocker-recovery-guide-plan.md [hunk 1]
```diff
@@ -253,6 +253,148 @@ This error might occur if you updated the firmware. As a best practice you shoul
...
 
+
+## Bitlocker Recovery Password screen
+During BitLocker recovery Windows can display **custom recovery message** and **password hints** identifying where password can be retrieved from. These two improvements can help user during BitLocker recovery.
+### Custom recovery message
+BitLocker group polices have been improved in Windows 10 Version 1511 by adding policy which allows for configuration of custom recovery message and URL on BitLocker Recovery screen. This policy cam be used to show address of BitLocker self-service recovery portal, IT internal website or phone number to helpdesk department.
```
Comment (illfated) [inline windows/security/information-protection/bitlocker/bitlocker-recovery-guide-plan.md:260]: May I suggest adding the spacing codestyle consistency as used in the rest of the document?
```suggestion
## Bitlocker Recovery Password screen

During BitLocker recovery Windows can display **custom recovery message** and **password hints** identifying where password can be retrieved from. These two improvements can help user during BitLocker recovery.

### Custom recovery message

BitLocker group polices have been improved in Windows 10 Version 1511 by adding policy which allows for configuration of custom recovery message and URL on BitLocker Recovery screen. This policy cam be used to show address of BitLocker self-service recovery portal, IT internal website or phone number to helpdesk department.
```
(Yes, I notice that it looks a bit more spread out than expected in source view, but this spacing allows for easier reading and modification of the source text at a later date. The resulting Rich Text or HTML page will show this spacing anyway.)

**Comment by Kellylorenebaker** [comment]
@rafals2 

Here are the [edits for your commit](https://github.com/MicrosoftDocs/windows-itpro-docs/pull/7063/commits/3883963058012494ac4fc3f1bdd6673b60cb987f). (See commit above.) The images all look fine from a content standpoint, but it would be good to check them again on staging when everything is finalized so we can make sure they render clearly. Please review and let me know if you have any questions about the edits.

Thanks!
Kelly.

**Comment by illfated** [comment]
**Current PR full file contents, in Rich text (web preview) format:**

- https://github.com/rafals2/windows-itpro-docs/blob/public/windows/security/information-protection/bitlocker/bitlocker-recovery-guide-plan.md

**Individual images previewed in "Rich diff" within the PR source view:**

- `bl-intune-custom-url.png`:
  https://github.com/MicrosoftDocs/windows-itpro-docs/pull/7063/files?short_path=406fb11#diff-406fb11371bffb57f1c310c37bc95ba6

- `bl-password-hint1.png`:
  https://github.com/MicrosoftDocs/windows-itpro-docs/pull/7063/files?short_path=695ead1#diff-695ead10c8ae5a6ca4bb200876328639

- `bl-password-hint2.png`:
  https://github.com/MicrosoftDocs/windows-itpro-docs/pull/7063/files?short_path=f080a72#diff-f080a72b4f81cdb6fa6bce613323d72c

- `rp-example1.PNG`:
  https://github.com/MicrosoftDocs/windows-itpro-docs/pull/7063/files?short_path=4c74589#diff-4c745899be338da7fa8a03ad3928f834

- `rp-example2.PNG`:
  https://github.com/MicrosoftDocs/windows-itpro-docs/pull/7063/files?short_path=7793f0a#diff-7793f0a61ca5cb124dc18336225f8489

- `rp-example3.PNG`:
  https://github.com/MicrosoftDocs/windows-itpro-docs/pull/7063/files?short_path=5ffcc61#diff-5ffcc610d1a1462799c9b7882d5c692c

- `rp-example4.PNG`:
  https://github.com/MicrosoftDocs/windows-itpro-docs/pull/7063/files?short_path=6815224#diff-6815224bba3bc601f3313bc2f4899892

- `rp-example5.PNG`:
  https://github.com/MicrosoftDocs/windows-itpro-docs/pull/7063/files?short_path=6815224#diff-6815224bba3bc601f3313bc2f4899892

**Comment by Kellylorenebaker** [comment]
@rafals2 These changes look good. The formatting of the Important note looks correct to me. It's doesn't render correctly in preview, so we'll need to verify on staging, which we don't have a link for here. I'll loop in @claydetels19 to move this to the next stage of publishing. Thanks!

**Pull request merged**